File: test/data/phenology/crop2ml/algo/cs/ShootNumber.cs

double oldCanopyShootNumber;
int emergedLeaves, shoots, i;
oldCanopyShootNumber = canopyShootNumber;
emergedLeaves = (int)Math.Max(1.0, Math.Ceiling(leafNumber - 1));
shoots = fibonacci(emergedLeaves);
canopyShootNumber = Math.Min(Shoots * sowingDensity, targetFertileShoot);
AverageShootNumberPerPlant = canopyShootNumber / sowingDensity;
if (canopyShootNumber != oldCanopyShootNumber)
{
	tilleringProfile.Add(canopyShootNumber - oldCanopyShootNumber);
}		
tillerNumber = tilleringProfile.Count;		
for (i = leafTillerNumberArray.Count; i < Math.Ceiling(leafNumber); i++)
{
	leafTillerNumberArray.Add(tillerNumber);
}

File: test/data/phenology/crop2ml/algo/java/ShootNumber.java
int emergedLeaves, shoots;
double oldCanopyShootNumber;
oldCanopyShootNumber = canopyShootNumber;
emergedLeaves = (int)Math.max(1, Math.ceil(leafNumber - 1));
shoots = fibonacci(emergedLeaves);
canopyShootNumber = Math.min(shoots * sowingDensity, targetFertileShoot);
averageShootNumberPerPlant = canopyShootNumber / sowingDensity;
if (canopyShootNumber != oldCanopyShootNumber)
{
	tilleringProfile.add(canopyShootNumber - oldCanopyShootNumber);
}		
tillerNumber = tilleringProfile.size();		
for (int i = leafTillerNumberArray.size(); i < Math.ceil(leafNumber); i++)
{
	leafTillerNumberArray.add(tillerNumber);
}
